TACC/Core-Styles
# remove pr title checker

## Overview / Related / Changes

Remove PR title checker action (originally from #176).

<sup>(because the PR labels are used to draft release and update changelog)</sup>

## Testing

- Check that PR Title Checker Action is not present on this PR.
- Check that this PR title (not conventional commit syntax) does not cause actions to fail.

## UI

N/A
